baselineProfile/build.gradle.kts

```
@@ -0,0 +1,52 @@
+import com.android.build.api.dsl.ManagedVirtualDevice
+
+plugins {
+    alias(libs.plugins.androidTest)
+    alias(libs.plugins.jetbrainsKotlinAndroid)
+    alias(libs.plugins.baselineprofile)
+}
+
+android {
+    namespace = "com.example.baselineprofile"
```

Should we update the module namespace?

```suggestion
    namespace = "com.github.libretube.baselineprofile"
```